packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -137,7 +148,7 @@ export function TableNG(props: TableNGProps) {
   const defaultRowHeight = getDefaultRowHeight();
   const defaultLineHeight = theme.typography.body.lineHeight * theme.typography.fontSize;
 
-  const HeaderCell: React.FC<HeaderCellProps> = ({ column, onSort, direction, justifyContent }) => {
+  const HeaderCell: React.FC<HeaderCellProps> = ({ column, field, onSort, direction, justifyContent }) => {
     const headerRef = useRef(null);
```

Comment:
```suggestion
    let filterableWithOverride = filterable;
    if (field.config.custom.filterable !== undefined) {
      filterableWithOverride = field.config.custom.filterable;
    }
    const headerRef = useRef(null);
```

Something like this and then update line 176 to use `filterableWithOverride`.

Reply:
This only makes the UI support filter overrides, but doesn't go the full way to making it fully functional.